.github/workflows/helm-charts.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.8.1
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.4.0
+        with:
+          charts_dir: helm
+          charts_repo_url: https://downloads.apache.org/flink/
```

Comment:
This should point to the github pages for flink-kubernetes-operator instead i think

Reply:
Hey @gyfora , 

that is true -- if we let to the defaults it points to github-pages (we do not have it currently here) and the idea was to use this as an alternative -- which will also host SNAPSHOTS (I got to know that it is only meant for releases and not also holding snapshots). So, I will update the chart-releaser-action to point to github-pages and let us also agree if we want to have gh-pages for SNAPSHOTs. My idea was to use only one hosting platform for both so that we do not need to change anything withing documentation but rather just use : 

```bash
helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-1.0-SNAPSHOT/
```
But I do also agree with the process and how that is done and thus we can also keep the documentation up-to-date when using snapshot refers to gh-pages and when using a proper release to `https://downloads.apache.org/flink/`